src/components/forms/formComponents.js

```diff
@@ -0,0 +1,387 @@
+      placeholder: placeholder || title,
+      value: formValue,
+      style: { ...styles.inputStyle, ...inputStyle },
+      disabled: disabled,
+      onChange: (event) => onFormInputChange(config, event.target.value),
+      onFocus: () => setError(),
+      onBlur: (event) => validateFormInput(config, event.target.value),
+      'aria-describedby': ariaDescribedby,
+    }),
+    errorMessage(error)
+  ]);
+};
+
+export const formInputMultiText = (config) => {
+  const {
+    id, title, disabled,
+    placeholder, ariaDescribedby,
+    inputStyle, onChange,
+    formValue, setFormValue, error, setError
+  } = config;
+
+  const pushValue = (element) => {
+    const value = element.value.trim();
+
+    if (!value || !validateFormInput(config, value)) {
+      return;
+    }
+    if (formValue.indexOf(value) !== -1) {
+      element.value = '';
+      return;
+    }
+
+    const formValueClone = cloneDeep(formValue);
+    formValueClone.push(value);
+    setFormValue(formValueClone);
+    onChange({key: id, value: formValueClone, isValid: true});
+    element.value = '';
+  };
+
+  const removePill = (index) => {
+    const formValueClone = cloneDeep(formValue);
+    formValueClone.splice(index, 1);
+    setFormValue(formValueClone);
+    onChange({key: id, value: formValueClone, isValid: true});
+  };
+
+  return div({}, [
+    div({
+      className: 'formControl-group flex-row',
+    }, [
+      input({
+        id,
+        type: 'text',
+        className: `form-control ${error ? 'errored' : ''}`,
+        placeholder: placeholder || title,
+        style: { ...styles.inputStyle, ...inputStyle },
+        disabled,
+        'aria-describedby': ariaDescribedby,
+        onKeyUp: (event) => event.code === 'Enter' ? pushValue(event.target) : setError(),
+        onFocus: () => setError()
+      }),
+      h(button, {
+        className: 'form-btn btn-xs',
+        type: 'button',
+        disabled,
+        style: {
+          marginTop: 0,
+          minWidth: 'fit-content'
+        },
+        onClick: () => pushValue(document.getElementById(id))
+      }, [
+        span({
+          className: 'glyphicon glyphicon-plus',
+          'aria-label': 'Add',
+          style: { margin: '0 8px' },
+          isRendered: !disabled
+        })
+      ])
+    ]),
+    errorMessage(error),
+    div({ className: 'flex-row', style: { justifyContent: null } },
+      formValue.map((val, i) => {
+        return h(button, {
+          key: val,
+          className: 'pill btn-xs',
+          type: 'button',
+          disabled,
+          onClick: () => removePill(i)
+        }, [
+          val,
+          span({
+            className: 'glyphicon glyphicon-remove',
+            style: { marginLeft: '8px' },
+            isRendered: !disabled
+          })
+        ]);
+      })
+    )
+  ]);
+};
+
+// Using react-select/creatable - Passing config directly through!
+export const formInputSelect = (config) => {
```

Comment:
For some reason, when I select a File Type, I'm not seeing the text displayed (but onChange is being called and formValue is being set)

Reply:
Fixed! Thank you!